server/database.go

```diff
@@ -701,21 +701,26 @@ func (s *Server) composeDatabaseRelationship(ctx context.Context, database *api.
 	}
 
 	rowStatus = api.Normal
-	labels, err := s.LabelService.FindDatabaseLabelList(ctx, &api.DatabaseLabelFind{
+	labelList, err := s.LabelService.FindDatabaseLabelList(ctx, &api.DatabaseLabelFind{
 		DatabaseID: &database.ID,
 		RowStatus:  &rowStatus,
 	})
 	if err != nil {
 		return err
 	}
 
-	if len(labels) > 0 {
-		labels, err := json.Marshal(labels)
-		if err != nil {
-			return err
-		}
-		database.Labels = string(labels)
+	// Each database instance is created under a particular environment.
+	// The value of bb.environment is identical to the name of the environment.
```

Comment:
I mean to explain the reason of returning here, since we already return instance inside database. and instance has access to the environment info.